public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -384,6 +384,28 @@ function Main (params) {
                 svl.labelCounter.set('Other', counter['Other']);
             });
 
+        /**
+         * Loads the labels onto the mini map when user start auditing if they are near any previously
+         * placed labels
+         */
+        svl.labelContainer.miniMapLabelsInRegion(
+            neighborhood.getProperty("regionId"),
+            function (result) {
+                var labelsArr = result.labels;
+                for (var i = 0; i < labelsArr.length; i++) {
+                    var imagePaths = util.misc.getIconImagePaths(),
+                        url = imagePaths[labelsArr[i].label_type].googleMapsIconImagePath;
```

Comment:
can you add have a `var` on every line that has a variable? I will add this to the style guide now :) https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide